platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-panel/log-panel.js

```diff
@@ -16,14 +16,35 @@
 import React from 'react'
 import LogViewer from '../log-viewer/log-viewer'
 
+const panelClass = () => {
```

Comment:
Ya, pretty much. In most situations, anonymous functions can be replaced with `(parameters) =>` just to be a bit less verbose. The only other thing it does is bind `this` for you, so you don't have to do the ugly `this = that` stuff to get proper functionality.

Mozilla has some pretty good docs on it if you're interested: [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)